parsl/monitoring/web_app/apps/workflows.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+import dash_html_components as html
+from parsl.monitoring.web_app.app import get_db, close_db
+from parsl.monitoring.web_app.utils import dataframe_to_html_table
+
+
+sql_conn = get_db()
+
+layout = html.Div(children=[
+    html.H1("Workflows"),
+    dataframe_to_html_table(id='workflows_table',
+                            dataframe=pd.read_sql_query("SELECT run_id, "
+                                                        "workflow_name, "
+                                                        "workflow_version, "
+                                                        "time_began, "
+                                                        "time_completed, "
+                                                        "tasks_completed_count, "
+                                                        "tasks_failed_count, "
+                                                        "user, "
+                                                        "host, "
+                                                        "rundir FROM workflows", sql_conn)
+                            .drop_duplicates(
+                                subset=['workflow_name', 'workflow_version'],
+                                keep='last')
```

Comment:
I think this is intending to keep only one of potentially duplicated workflow_name/_version records here.

Does it matter which one is kept?

I think the SQL statement in general doesn't guarantee any particular order of results, so keeping the last one is sort-of like quasi-randomly picking one.